packages/alfa-css/src/syntax/declaration.ts

```diff
@@ -195,11 +196,13 @@ export namespace Declaration {
           const result = consume(Slice.of(value));
 
           if (result.isOk()) {
-            declarations.push(result.get()[1]);
+            // the check ensures the result is Ok
+            declarations.push(result.getUnsafe()[1]);
```

Comment:
**Thought:** This is weird 🤔 `.isOk` should correctly narrow the type.

Reply:
Indeed it did, must have been a mouse slip 😅